AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -2650,6 +2650,53 @@ export interface IEndpointRequest {
     defence_advocate_username?: string | undefined;
 }
 
+export enum LinkedParticipantType {
```

Comment:
I think this is a duplication.

Reply:
this file (api-client.ts) is auto generated